src/plugineventview.h

```diff
@@ -43,6 +43,7 @@ class PluginEventView : public QObject, public QDBusContext
     Q_NOREPLY void Destroy();
     virtual QList<QVariantMap> NextPage() = 0;
     virtual bool IsValid() const;
+    virtual int TotalCount();
```

Comment:
Since the implementation of this function does nothing useful and this is already an abstract class, you might want to make the pure virtual `virtual int TotalCount() = 0;` instead. This will ensure that any implementations of PluginEventView implement the TotalCount function.